### PR TITLE
add v2.3.0 related descriptions in changeLog

### PR DESCRIPTION
**Pull request by cyjseagull**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,41 @@
 ### v2.2.0
 
+(2020-03-24)
+
+**新特性**
+
+- 链上支持同态加密功能
```
Comment (bxq2011hust) [inline Changelog.md]:
```suggestion
- 新增同态预编译合约
```

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+- 链上支持群签名验证和环签名验证
```
Comment (wheatli) [inline Changelog.md]:
```suggestion
- 支持群签名和环签名
```

File: Changelog.md (hunk 1 above)
Comment (wheatli) [inline Changelog.md]:
```suggestion
- 支持同态加密
```

File: Changelog.md [hunk 3]
```diff
@@ -1,5 +1,41 @@
...
+- 基于PBFT共识算法，实现一种新型的共识算法RPBFT，尽量减少节点规模对共识算法的影响
```
Comment (wheatli) [inline Changelog.md]:
```suggestion
- 实现一种新型的共识算法RPBFT，尽量减少节点规模对共识算法的影响
```

File: Changelog.md (hunk 1 above)
Comment (cyjseagull) [reply Changelog.md]: Fixed.

File: Changelog.md (hunk 2 above)
Comment (cyjseagull) [reply Changelog.md]: Fixed.

File: Changelog.md (hunk 1 above)
Comment (cyjseagull) [reply Changelog.md]: Fixed.

File: Changelog.md (hunk 3 above)
Comment (cyjseagull) [reply Changelog.md]: Fixed.

**Pull request merged**